app/views/products/_form.html.erb

```diff
@@ -31,6 +31,21 @@
     <%= form.text_field :price %>
   </div>
 
+  <div>
+    <%= form.label :discount_price, style: "display: block" %>
+    <%= form.text_field :discount_price %>
```

Comment:
please permit these in the controller also

Reply:
Done